Source/tiff.m

```diff
@@ -71,9 +71,9 @@
 #include <math.h>
 #include <stdlib.h>
 #include <string.h>
-#ifndef __WIN32__
+#if defined(HAVE_UNISTD_H)
 #include <unistd.h>		/* for L_SET, etc definitions */
-#endif /* !__WIN32__ */
+#endif /* !HAVE_UNISTD_H */
```

Comment:
The acclamation mark is wrong here, the old condition was negated the new isn't. Otherwise this change looks fine. If I manage to change this inline, I will merge right away.

Reply:
Sorry, missed that. Thanks for fixing & merging, @fredkiefer!